### PR TITLE
Capistrano

### LINKED ISSUE #37: Capistrano
Add capistrano configuration for deployment

### PR DESCRIPTION
**Pull request by josephbridgwaterrowe**
Closes #37

### CODE REVIEW

**Comment by josephbridgwaterrowe** [comment]
@michaelachrisco please review.

File: config/deploy.example.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+set :keep_releases, 5
+set :linked_files, %w{config/application.yml config/database.yml}
+set :linked_dirs,
+    %w{bin log tmp/pids tmp/cache tmp/sockets vendor/bundle public/system}
+set :normalize_asset_timestamps,
+    %w{public/images public/javascripts public/stylesheets}
+set :repo_url, ''
+
+namespace :deploy do
+  # https://github.com/mperham/sidekiq/wiki/Signals
```
Comment (michaelachrisco) [inline config/deploy.example.rb]: Are you planning on using sidekiq?

File: lib/capistrano/tasks/sidekiq.rake [hunk 2]
```diff
@@ -0,0 +1,23 @@
+namespace :sidekiq do
+  desc 'Restart application'
+  # task :quiet do
```
Comment (michaelachrisco) [inline lib/capistrano/tasks/sidekiq.rake]: Never mind :smile: I see.

**Comment by michaelachrisco** [comment]
You have cap tasks and references to Sidekiq, but I don't see the Gem included in your Gemfile?

**Comment by josephbridgwaterrowe** [comment]
I'm not, I wanted to keep the tasks as a precaution :smile:

If you don't feel good about this I would be happy to remove.

**Comment by michaelachrisco** [comment]
Its just a bit strange referencing a gem not included in a repository. I would suggest removing them until you need them and adding them in later in its own branch. YAGNI and all that.

**Comment by josephbridgwaterrowe** [comment]
K, will remove.

**Comment by michaelachrisco** [comment]
:+1:  Rebase and I will merge.

**Pull request merged**